xbmc/video/test/TestVideoInfoScanner.cpp

```diff
@@ -37,10 +42,11 @@ static const TestEntry TestData[] = {
     {"foo.Part.3.mkv", {{1, 3}}, {{1, 3}}},
     {"foo.E03.mkv", {{1, 3}}, {{1, 3}}},
     {"foo.2009.E03.mkv", {{1, 3}}, {{1, 3}}},
-    // multi-episode
+    // multi-episode (not ranges)
     {"The Legend of Korra - S01E01-02 - Welcome to Republic City & A Leaf in the Wind.mkv",
      {{1, 1}, {1, 2}},
      {{1, 1}, {1, 2}}},
+    {"foo s01e01-episode1.title-s01e02-episode2.title.mkv", {{1, 1}, {1, 2}}, {{1, 1}, {1, 2}}},
```

Comment:
As mentioned in an earlier comment, it's difficult to impossible to make this one work reliably, the episode title could be a number that will be interpreted as an episode number. Impact was low before episode ranges, it's higher now. Low priority as far as I am concerned.

Reply:
Some examples that are not so easy to deal with for that file name format:

| show | ep | ep title |
|---|---|---|
| 24 | S01E01 | 12:00 A.M. - 1:00 A.M. |
| 24 |S01E02 | 1:00 A.M. - 2:00 A.M. |
| Casual | S02E06 | 100 Cows |
| Miracle Workers | S01E02 | 13 Days |
| Flashforward | S01E03 | 137 Sekunden |